datacenter/dtr/2.2/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -37,17 +43,30 @@ command creates a backup of DTR:
 
 These files are added to a tar archive, and the result is streamed to stdout.
 
+Things DTR's backup command doesn't back up:
```

Comment:
Write as if you were explaining this to me face to face. In this case I bet you'd say

DTR doesn't backup: